verification/curator-service/src/model/source.ts

```diff
@@ -15,17 +15,68 @@ const originSchema = new mongoose.Schema({
     },
 });
 
+interface Field {
+    name: string;
+    regexp: string;
+}
+
+const fieldSchema = new mongoose.Schema({
+    url: {
+        type: String,
+        required: 'Enter a dotted.field.name',
+    },
+    regexp: {
+        type: String,
+        required: 'Enter a field extraction regexp',
+    },
+});
+
+interface Parsing {
+    fields: Array<Field>;
+}
+
+const parsingSchema = new mongoose.Schema({
+    fields: {
```

Comment:
nit: can this just be `fields: [fieldSchema],`?

Reply:
indeed, thanks